pegasus/routes/v2_user_routes.rb

```diff
@@ -3,7 +3,11 @@
   dont_cache
   forbidden! unless dashboard_user
   content_type :json
-  JSON.pretty_generate(dashboard_user.slice_keys(:id, :name))
+  result = dashboard_user.slice_keys(:id, :name, :admin)
+  result[:owned_sections] = DASHBOARD_DB[:sections].
```

Comment:
Oh, yeah, I misspoke. Yeah, it feels like `dashboard_user.owned_sections` should be easy to get to from here.